Formula/svtplay-dl.rb

```diff
@@ -5,6 +5,7 @@ class SvtplayDl < Formula
   homepage "https://svtplay-dl.se/"
   url "https://files.pythonhosted.org/packages/0c/19/b590fd5ca34ca7cc0c2aa9e7d935d5c1330ef5a99e93136ce040667adac4/svtplay-dl-2.4.tar.gz"
   sha256 "bfe3b61c3294e01ee17fa670a10204e8369393fbc2d9f0c917fdc73026355761"
+  revision 2
```

Comment:
This should be bumped by 1 at a time, so `revision 1`. (Consider the absence of a `revision` block to be like `revision 0`).

Reply:
ahh okey.  i fixed that. thanks for the review <3